src/components/input/input.ts

```diff
@@ -75,10 +78,22 @@ import { Platform } from '../../platform/platform';
 @Component({
   selector: 'ion-input',
   template:
-    '<input [type]="type" [(ngModel)]="_value" (blur)="inputBlurred($event)" (focus)="inputFocused($event)" [placeholder]="placeholder" class="text-input" [ngClass]="\'text-input-\' + _mode">' +
+    '<input [type]="type" [(ngModel)]="_value" (blur)="inputBlurred($event)" ' +
+    '(focus)="inputFocused($event)" [placeholder]="placeholder" ' +
+    'class="text-input" [ngClass]="\'text-input-\' + _mode" ' +
+    '[attr.autocomplete]="autocomplete" [attr.autocorrect]="autocorrect"' +
+    '[attr.autocapitalize]="autocapitalize" [autofocus]="autofocus"' +
+    '[attr.list]="list" [attr.max]="max" [attr.maxlength]="maxlength"' +
+    '[attr.min]="min" [attr.minlength]="minlength" [readonly]="readonly"' +
```

Comment:
1. Fair question; short answer: `md-input`. I know I can query [DomElementSchemaRegistry](https://github.com/angular/angular/blob/61129fa12d34ef35996eec85a48295f873b21dde/modules/%40angular/compiler/src/schema/dom_element_schema_registry.ts) to verify this, and I'm getting the impression you may be right it's saying something different there, so I am getting curious about `md-input`'s implementation there as well.
2/3. Got it. I noticed `disabled` was added for input but unused, so added that too. Also, TextArea.
4. @jelbourn: I haven't verified, but if autofocus is not supported by ng2 as Manu says, would you mind commenting on its inclusion in Material2's `md-input`?

To properly finish the test (and query `DomElementSchemaRegistry`) I guess I'm gonna need to find a way to run it, but `gulp test` gives me this; bit stumped.

```
[09:11:20] Starting 'compile.karma'...

/PATH/ionic/node_modules/.bin/ngc:2
basedir=$(dirname "$(echo "$0" | sed -e 's,\\,/,g')")
          ^^^^^^^
SyntaxError: missing ) after argument list
```